src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -158,6 +158,11 @@ private Task PackageInitializationBackgroundThreadAsync(IProgress<ServiceProgres
 
         var settingsEditorFactory = this.ComponentModel.GetService<SettingsEditorFactory>();
 
+        // Misc workspace has to be up and running by the time our package is usable so that it can track running
+        // doc events and appropriately map files to/from it and other relevant workspaces (like the
+        // metadata-as-source workspace).
+        var miscellaneousFilesWorkspace = this.ComponentModel.GetService<MiscellaneousFilesWorkspace>();
```

Comment:
This could actually be removed entirely, and the comment transferred to the use in AbstractPackage`2. If in the scenario where the Roslyn package loaded _without_ one of the language specific packages being loaded...it won't do anything. Fundamentally it'll need that RegisterMiscellaneousFilesWorkspaceInformation method to get called so it actually can do the last bit.

Reply:
Done!